tests/storage/hpp/test_hostpath.py

```diff
@@ -364,7 +365,7 @@ def test_hpp_specify_node_immediate(
         source="http",
         dv_name="cnv-3228",
         namespace=namespace.name,
-        url=f"{get_test_artifact_server_url()}{Images.Rhel.RHEL8_0_IMG}",
+        url=f"{get_test_artifact_server_url()}{LATEST_RHEL_IMAGE}",
```

Comment:
Actually, you can do this:
```suggestion
        url=f"{get_test_artifact_server_url()}{py_config["latest_rhel_os_dict"]["image_path"]}",
```